cdc/generation.cc

```diff
@@ -292,6 +295,22 @@ future<utils::chunked_vector<mutation>> get_cdc_generation_mutations(
         co_await coroutine::maybe_yield();
     }
 
+    total_size_estimate += size_estimate;
+
+    // Copy mutations n times, where n is picked so that the memory size of all mutations together exceeds `max_command_size`.
+    utils::get_local_injector().inject("cdc_generation_mutations_replication", [&res, total_size_estimate, mutation_size_threshold] {
+        utils::chunked_vector<mutation> new_res;
+
+        size_t number_of_copies = (mutation_size_threshold / total_size_estimate + 1) * 2;
+        for (size_t i = 0; i < number_of_copies; ++i) {
+            for (auto r : res) {
+                new_res.push_back(r);
+            }
```

Comment:
I know this is test code, but... you have an unnecessary copy here that you could remove. Either:

- change the type of `r` to `const auto&`,
- `std::move(r)` when `pushing_back`,
- or replace the innermost loop with `std::copy`.

Reply:
Changed to `std::copy`.